guides/common/modules/proc_creating-custom-provisioning-snippets.adoc

```diff
@@ -1,25 +1,24 @@
 [id="Creating_Custom_Provisioning_Snippets_{context}"]
 = Creating custom provisioning snippets
 
-Custom provisioning snippets allow you to execute custom code during host provisioning.
-You can run code before and/or after the provisioning process.
+You can execute custom code before and/or after the host provisioning process.
 
 .Prerequisites
 * Depending on your provisioning template, multiple custom snippet hooks exist which you can use to include custom provisioning snippets.
-Ensure that you check your provisioning template first to verify which custom snippets you can use.
+Check your provisioning template first to verify which custom snippets you can use.
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *Templates* > *Provisioning Templates* and click *Create Template*.
 . In the *Name* field, enter a name for your custom provisioning snippet.
-The name has to start with the name of a provisioning template that supports including custom provisioning snippets:
-** Append ` custom pre` to the name of a provisioning template to run code before provisioning a host.
-** Append ` custom post` to the name of a provisioning template to run code after provisioning a host.
+The name must start with the name of a provisioning template that supports including custom provisioning snippets:
+** Append `custom pre` to the name of a provisioning template to run code before provisioning a host.
+** Append `custom post` to the name of a provisioning template to run code after provisioning a host.
```

Comment:
I find the name requirements (begin with template name, append `custom pre` or `custom post`) confusing. I recommend you use examples from the CLI procedure: `My_Template_Name_custom_pre`

Reply:
The original template name depends on the default template that is used for REX. There is no wiggle room for users; it has to 100% match the name of the template + ` custom pre`/` custom post`.

Added an example that works for Foreman/Katello, orcharhino, and Satellite.

Does this work for you?